src/createServer.js

```diff
@@ -13,7 +13,7 @@ module.exports = createServer
 
 function createServer (options = {}) {
   const {
-    host = '0.0.0.0',
+    host,
```

Comment:
This looks good to me. I don't see the point of doing `host = undefined`. It is redundant in my opinion.